File: Products/sample-android/app/src/main/java/mockifer/example/android/cats/Cat.java
package mockifer.example.android.cats;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Cat implements Serializable {
    @SerializedName("id") public String id;
    @SerializedName("name") public String name;
    @SerializedName("age") public int age;
    @SerializedName("image") public String image;
}

File: Support/samples/sample-android/app/src/main/java/mockifer/example/android/cats/Cat.kt
package mockifer.example.android.cats

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Cat(@SerializedName("id") var id: String,
          @SerializedName("name") var name: String,
          @SerializedName("age") var age: Int,
          @SerializedName("image") var image: String
) : Serializable
